src/rmp/src/zero_slack_strategy.cpp

```diff
@@ -54,32 +57,38 @@ void ZeroSlackStrategy::OptimizeDesign(sta::dbSta* sta, utl::Logger* logger)
 
   sta::dbNetwork* network = sta->getDbNetwork();
 
+  std::vector<sta::Vertex*> candidate_vertices = GetNegativeEndpoints(sta);
+
+  if (candidate_vertices.empty()) {
+    logger->warn(
```

Comment:
info is sufficient as nothing has gone wrong.  ("Warnings should be used to indicate atypical runtime conditions that may affect quality, but not correctness, of the output.")